hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md

```diff
@@ -600,12 +602,115 @@ SystemInfo
 3 passing (287ms)
 ```
 
+## Within a Docker network
+
+You may not have used Docker up to this point. The following paragraphs acquaint you with a Docker _user-defined bridged network_. If you plan on using Docker Compose at a later stage, having a first taste of such networks is beneficial. Docker Compose can be used to orchestrate and launch separate containers in order to mimic a production setup. If you think this could eventually be useful, you should complete this section.
+
+Install [Docker](https://docs.docker.com/get-docker/).
+
+To run the checkers chain with Ignite CLI you have the choice of two Docker images:
+
+1. You can use the one published by Ignite themselves, for version `0.22.1`: [`ignitehq/cli:0.22.1`](https://hub.docker.com/layers/ignitehq/cli/0.22.1/images/sha256-8e2f353f943227488f966dd02558b718766a17dd8b611bccd2789facdceef0cf). This may be faster the first time you run it, but can become annoying if you plan on doing it many times as it will download the Go dependencies every time.
+2. You can build it yourself from the checkers [`Dockerfile-ubuntu`](https://github.com/cosmos/b9-checkers-academy-draft/blob/main/Dockerfile-ubuntu), with the command:
+
+    ```sh
+    $ docker build -f Dockerfile-ubuntu . -t checkers_i
+    ```
+
+    This is the preferred method if you plan on using the image many times, as it downloads all Go dependencies once.
```

Comment:
is this tab intentional or should i just be a space

Reply:
It is intentional, in order to have this paragraph part of the indented numbered list item 2.